### PR TITLE
fix home page typo

### PR DESCRIPTION
**Pull request by sheykerkup**
addresses odk-x/tool-suite-X#207

#### What is included in this PR?
Line edits in documentation homepage

### CODE REVIEW

File: odkx-src/index.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ ODK-X documentation
 
 .. _odk-2-introduction:
 
-The ODK-X Tool Suite is free and open-source software for collecting, managing, and using data in resource-constrained environments.
+The ODK-X Tool Suite is free and open-source software for collecting, managing and using data in resource-constrained environments.
```
Comment (elmps2018) [inline odkx-src/index.rst:10]: Per the style guide, the comma should be here, see: https://docs.odk-x.org/docs-style-guide/#serial-comma (I know different countries/styles do or don't use the penultimate/Oxford comma so it's a confusing one).

File: odkx-src/index.rst [hunk 2]
```diff
@@ -38,7 +38,7 @@ ODK-X tools enable you to open and edit other surveys with links back to the ori
 
 View data on device
 """""""""""""""""""
-Investigate and visualize entire datasets directly on the device through graphical, map, and tabular displays and through filtered views.
+Investigate and visualize entire datasets directly on the device through graphical, map, tabular displays and filtered views.
```
Comment (elmps2018) [inline odkx-src/index.rst:41]: add comma after tabular displays.

**Review by elmps2018 — CHANGES_REQUESTED** [review]

File: odkx-src/index.rst (hunk 1 above)
Comment (sheykerkup) [reply odkx-src/index.rst:10]: Ok i see

File: odkx-src/index.rst (hunk 2 above)
Comment (sheykerkup) [reply odkx-src/index.rst]: Changes done.

**Review by elmps2018 — APPROVED** [review]

**Review by elmps2018 — APPROVED** [review]

**Pull request merged**